code/obj/artifacts/artifact_items/power_cell.dm

```diff
@@ -40,7 +40,7 @@
 
 /datum/artifact/powercell
 	associated_object = /obj/item/cell/artifact
-	rarity_class = 1
+	rarity_class = 2 // modified from 1 as part of art tweak
```

Comment:
I don't really think adding a comment for each such change is necessary, it should be apparent from the git history?
But it's not really a problem either.

Reply:
Yeah the some of the comments are superfluous and were because I was going to share the code with Lurk for testing but I had no concept of doing that with git until I had already written them.